README.md

```diff
@@ -1,72 +1,44 @@
-described in the ASTM remote ID standard.  This flexible and distributed
-system is used to connect multiple USSs operating in the same general
-area to share information while protecting operator and consumer
-privacy. The system is focused on facilitating communication amongst
-actively operating USSs without details about UAS operations stored or
-processed in the DSS.
 
-## Concepts
+This repository contains the implementation of the Discovery and Synchronization Service (DSS) and a monitoring framework to test UAS Service Suppliers (USS). See the [InterUSS website](https://interuss.org) for background information.
 
-Conceptual background on the DSS and services it supports may be found [here](concepts.md).
+**Standards and Regulations**
 
-## Simplified architecture
+The DSS implementation and associated monitoring tools comply with the following standards and regulations:
 
-### Overview
-![Simplified architecture diagram](assets/generated/simple_architecture.png)
+- [ASTM F3411-19](https://www.astm.org/Standards/F3411.htm): Remote ID. [See OpenAPI interface](./interfaces/uastech/standards/remoteid)
+- [ASTM WK63418](https://www.astm.org/DATABASE.CART/WORKITEMS/WK63418.htm): UAS Traffic Management (UTM) UAS Service Supplier (USS) Interoperability Specification. [See OpenAPI interface](./interfaces/astm-utm)
 
-A "DSS Region" consists of one or more DSS instances sharing the same
-DSS Airspace Representation (DAR) by forming a single CockroachDB
-cluster.  In the simplified diagram above, two DSS instances share the
-same DAR via CRDB certificates and configuration which means the two
-HTTPS frontends may be used interchangeably.  USS 1 chooses to use only
-instance 1 while USS 2 uses both instances for improved resilience to
-failures.
+U-Space specific:
 
-### HTTPS frontend
+- [COMMISSION IMPLEMENTING REGULATION (EU) 2021/664](https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32021R0664&from=EN#d1e32-178-1) *(work in progress)*
 
-Serves as an HTTPS gateway to the business logic, translating between
-HTTPS request and gRPC to allow users to communicate to the DSS via
-simple HTTPS calls. This code is currently generated via
-[grpc-gateway](https://github.com/grpc-ecosystem/grpc-gateway), and does
-not do much other than translation.  See the [API specification
-here](https://tiny.cc/dssapi_rid).
+## Discovery and Synchronization Service (DSS)
 
-### gRPC backend
+The DSS is a simple and open service used by separate USSs, often in different organizations, to communicate information about UAS operations and coordinate with each other. This service is described in the ASTM remote ID standard. This flexible and distributed system is used to connect multiple USSs operating in the same general area to share information while protecting operator and consumer privacy. The system is focused on facilitating communication amongst actively operating USSs without details about UAS operations stored or processed in the DSS.
```

Comment:
nit: Can this line be easily broken into multiple lines?  It makes future diffs easier to view (same comment for other lines beyond ~110 characters)